guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
   - [IO Specification](#io-specification)
   - [Local Execution](#local-execution)
-  - [Containerization](#containerization)
   - [Compatible with Kubernetes](#compatible-with-kubernetes)
   - [OCI Compliance](#oci-compliance)
-  - [Facilitate GPU Acceleration](#facilitate-gpu-acceleration)
-- [Architecture & Design](#architecture--design)
-- [Description](#description)
+  - [Hosting Environment](#hosting-environment)
+- [Description](#description-1)
 - [Application](#application)
 - [Manifests](#manifests)
   - [Application Manifest](#application-manifest)
   - [Package Manifest](#package-manifest)
-- [Executor](#executor)
-  - [Initial Conditions / Environment Variables](#initial-conditions)
-  - [Manifest Export](#manifest-export)
+- [Supplemental Application Files](#supplemental-application-files)
+  - [Container Behavior and Interaction](#container-behavior-and-interaction)
   - [Table of Important Paths](#table-of-important-paths)
-- [Package Layout Diagram](#layout-diagram)
+- [Package Layout Diagram](#package-layout-diagram)
 
+<!-- /code_chunk_output -->
 
 ## Overview
 
-This proposal documents the specification of the initial version of the MONAI Application Package (MAP).
-
+This proposal documents the specification of the MONAI Application Package (MAP).
 
 ### Goal
 
-The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP and how it can be
+The goal of this proposal is to provide the structure of a MAP, define the purpose of a MAP, and how it can be
 interacted with, and the required and optional components of a MAP.
 
+### Assumptions, Constraints, Dependencies
 
-### Assumptions
-
-The following are a set of assumptions related to MAP execution, inspection, and general usage.
+The following is a set of assumptions about MAP execution, inspection, and general usage.
 
-- Containerized applications will be based on Linux x86_64 (AMD64).
+- Containerized applications will be based on Linux x64 (AMD64) and/or ARM64 (aarch64).
 
-- Containerized application's host environment will be based on Linux x86_64 (AMD64) with container support.
+- Containerized application's host environment will be based on Linux x64 (AMD64) and/or ARM64 (aarch64) with container support.
 
-- Developers expect local execution of their applications to behave identically to execution of the containerized version.
+- Developers expect the local execution of their applications to behave identically to the execution of the containerized version.
 
 - Developers expect local execution of their containerized applications to behave identically to execution in deployment.
 
 - Developers and operations engineers want the application packages to be self-describing.
 
-- Application packages might not be developed using the MONAI Application SDK.
+- Applications might not be developed using the Holoscan SDK or an SDK based on it, e.g., MONAI Deploy App SDK.
```

Comment:
It's not clear quite what the stipulation is here. Are we trying to say that Application packages don't have to be developed using the Holoscan SDK and that the Monai Deploy Add SDK is an example of an SDK that is not based on Holoscan?

Reply:
Simplifying it to `Applications might not be developed using the Holoscan SDK or the MONAI Deploy App SDK.`